Solutions/CohesitySecurity/Tests/helios.py

```diff
@@ -96,32 +96,44 @@ def get_days_ago_timestamp(days_ago):
     return [jsObj["id"] for jsObj in response.json()]
 
 
-def get_recoveries(cluster_id: str, api_key: str) -> dict:
+def get_recoveries(cluster_id: str, api_key: str, start_time_usecs=None, end_time_usecs=None) -> dict:
     """
     This function sends a GET request to the Helios API to get a list of all
     recoveries for a given Cohesity cluster.
 
     Args:
         cluster_id (str): The cluster ID of the Cohesity cluster.
         api_key (str): The API key to authenticate the request.
+        start_time_usecs (str): The start time for the recoveries (in microseconds).
```

Comment:
Consider making them integers. And convert them as strings within the function. Also, please mention that if 'None' is passed, those values will be not in the request.

Reply:
done